Public/Src/Cache/ContentStore/Vfs/VfsProgram.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BuildXL.ToolSupport;
+
+namespace BuildXL.Cache.ContentStore.Vfs
+{
+    using System.Diagnostics.ContractsLight;
+    using BuildXL.Cache.ContentStore.Interfaces.FileSystem;
+    using static CommandLineUtilities;
+
+    public class VfsProgram : ToolProgram<VfsCasConfiguration>

```

Comment:
>VfsProgram [](start = 17, length = 10)

Should this be inside ContentStoreApp.exe instead?

Reply:
Still debating that with myself. Will discuss with team.

---
In reply to: [280871326](https://github.com/Microsoft/BuildXL/pull/249#discussion_r280871326) [](ancestors = 280871326)